_gsocblogs/2022/blog_CERNBOX_KarthikSundar.md

```diff
@@ -0,0 +1,38 @@
+---
+project: CERNBOX
+title: OCM WebDAV Service
+author: Karthik Sundar
+photo: blog_authors/KarthikSundar.jpg
+date: 22.07.2022
+year: 2022
+layout: blog_post
+logo: cernbox-logo.png
+intro: |
+  File Sharing through a OCM WebDAV Service, a Progress Report.
```

Comment:
Please use an actual introduction in a sentence, this is what people see first before reading your blog.

Reply:
Yeah, i have changed the intro.